<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="Guide to all things human 'omics data sharing in Australia.">
    <meta name="keywords" content="">
    <meta property="og:title" content="Human 'Omics Data Sharing Field Guide" />
    <meta property="og:description" content="Guide to all things human 'omics data sharing in Australia." />
    <meta property="og:image" content="//mshadbolt.github.io/human-omics-data-sharing-field-guide/assets/img/apple-touch-icon.png" />
    <meta name="apple-mobile-web-app-title" content="Human 'Omics Data Sharing Field Guide">
    <meta name="msapplication-TileColor" content="#">
    <meta name="theme-color" content="#">
    <title>References | Human 'Omics Data Sharing Field Guide</title>
    <!-- Syntax highlighting  -->
    <link rel="stylesheet" href="/human-omics-data-sharing-field-guide/assets/css/syntax.css">
    <!-- Country flags  -->
    <link rel="stylesheet" href="/human-omics-data-sharing-field-guide/assets/css/flag-icon.min.css">
    <!-- Font Awesome -->
    <link rel="stylesheet" type="text/css" href="/human-omics-data-sharing-field-guide/assets/css/all.min.css">
    <!-- Main css file -->
    <link rel="stylesheet" href="/human-omics-data-sharing-field-guide/assets/css/main.css">
    <!-- Main font(s) -->
    <!-- JavaScript  -->
    <script src="/human-omics-data-sharing-field-guide/assets/js/jquery.min.js"></script>
    <script src="/human-omics-data-sharing-field-guide/assets/js/lunr.min.js"></script>
    <script src="/human-omics-data-sharing-field-guide/assets/js/bootstrap.bundle.min.js"></script>
    <script src="/human-omics-data-sharing-field-guide/assets/js/anchor.min.js"></script>
    <script src="/human-omics-data-sharing-field-guide/assets/js/toc.js"></script>
    <script src="/human-omics-data-sharing-field-guide/assets/js/jquery.navgoco.js"></script>
    <script src="/human-omics-data-sharing-field-guide/assets/js/main.js"></script>
    <script src="/human-omics-data-sharing-field-guide/assets/js/search.js"></script>
    <!-- favicon -->
    <link rel="apple-touch-icon" sizes="180x180" href="/human-omics-data-sharing-field-guide/assets/img/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="/human-omics-data-sharing-field-guide/assets/img/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/human-omics-data-sharing-field-guide/assets/img/favicon-16x16.png">
    <link rel="manifest" href="/human-omics-data-sharing-field-guide/assets/img/site.webmanifest">
    <link rel="mask-icon" href="/human-omics-data-sharing-field-guide/assets/img/safari-pinned-tab.svg" color="#">
    <link rel="shortcut icon" href="/human-omics-data-sharing-field-guide/assets/img/favicon.ico">
    <!-- Schema.org metadata -->
    <script type="application/ld+json">
    {
        "@context": "https://schema.org",
        "@type": "WebSite",
        "name": "Human 'Omics Data Sharing Field Guide",
        "url": "https://mshadbolt.github.io",
        "about": "Guide to all things human 'omics data sharing in Australia."
    }
</script>

</head>

<body>
    <!-- Navigation -->
<header>
    <a class="visually-hidden-focusable" href='#main'>Skip to content</a>
    <a class="visually-hidden-focusable" href='#footer'>Skip to footer</a>
    <nav class="navbar navbar-light navbar-expand-lg mb-3 mb-lg-5">
        <div class="container">
            <a class="navbar-brand" href="/human-omics-data-sharing-field-guide/"><img class="me-3 img-fluid" alt="Human 'Omics Data Sharing Field Guide logo" src="/human-omics-data-sharing-field-guide/assets/img/main_logo.svg"><span class="me-0 me-lg-3">Human 'Omics Data Sharing Field Guide</span></a>
            <button class="navbar-toggler text-primary" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
                <i title="navbar-toggler" class="fa-solid fa-bars"></i>
            </button>
            <div class="collapse navbar-collapse justify-content-end" id="navbarCollapse">
                <ul class="navbar-nav">
                    <!-- navigation entries -->
                    <li class="nav-item ms-0 ms-lg-3 mt-2 mt-lg-0">
                        <a class="nav-link ps-2" href="/human-omics-data-sharing-field-guide/index">Home</a>
                    </li>
                    <li class="nav-item ms-0 ms-lg-3 mt-2 mt-lg-0">
                        <a class="nav-link ps-2 active" aria-current="page" href="/human-omics-data-sharing-field-guide/about">About</a>
                    </li><li class="nav-item ms-0 ms-lg-3 mt-2 mt-lg-0 nav-break bg-dark opacity-25"></li>
                    <li class="nav-item ms-0 ms-lg-3 mt-2 mt-lg-0">
                        <a class="nav-link ps-2 d-flex align-items-center" href="https://github.com/mshadbolt/human-omics-data-sharing-field-guide"><i class="fa-brands fa-github me-2"></i>GitHub</a>
                    </li>
                    <!--start search-->
                    <li class="nav-item ms-0 ms-lg-3 my-2 my-lg-0">
                        <div class="position-relative">
                            <form role="search" class="input-group">
                                <span class="input-group-text" id="search-label"><i class="fa-solid fa-magnifying-glass"></i></span><input type="search" id="search-input" class="search-input form-control" tabindex="0" placeholder="Search Human 'Omics Data Sharing Field Guide" aria-label="Search Human 'Omics Data Sharing Field Guide" autocomplete="off">
                            </form>
                            <div id="search-results" class="search-results"></div>
                        </div>
                    </li>
                    <!--end search-->
                </ul>
            </div>
        </div>
        <!-- /.container -->
    </nav>
</header>

    <!-- Page Content -->
    <div class="container min-vh-100">
        <!-- Content Row -->
        <div class="row gx-0 gx-lg-5">
            <!-- Sidebar Column -->
            <div class="col-lg-3">
                <button class="btn text-start d-lg-none collapsed sidebar-collapse w-100 mb-3" type="button" data-bs-toggle="collapse" data-bs-target="#side-nav" aria-controls="side-nav" aria-expanded="true" aria-label="Toggle navigation">
    About menu <i title="navbar-toggler" class="fa-solid fa-bars ms-2"></i>
</button>
<nav class="collapse" id="side-nav" aria-label="Side navigation">
    <a class="btn hover-primary text-start d-block mb-1 sidebar-title" href="/human-omics-data-sharing-field-guide/about">About </a>
    <ul class="list-unstyled mb-4 mb-lg-0">
        <li>    
            <a class="btn hover-primary text-start d-block mb-1" href="/human-omics-data-sharing-field-guide/contributors">Contributors</a>
        </li>
        <li>    
            <a class="btn hover-primary text-start d-block mb-1" href="/human-omics-data-sharing-field-guide/contribute">Contribute</a>
        </li>
        <li>    
            <a class="btn hover-primary text-start d-block mb-1" href="/human-omics-data-sharing-field-guide/acknowledgements">Acknowledgements</a>
        </li>
        <li class="active">    
            <a class="btn hover-primary text-start d-block mb-1" href="/human-omics-data-sharing-field-guide/references">References</a>
        </li>
    </ul>
    <script>$("li.active").parents('li').toggleClass("active parent");</script>
</nav>

            </div>
            <!-- Content Column -->
            <div class="col-lg-9">
                <main id="main" class="mb-5">
    <h1>References
    
<div id="edit-me" class="btn-group">
    <a role="button" data-bs-toggle="tooltip" title="Propose changes to this page on GitHub"  href="https://github.com/mshadbolt/human-omics-data-sharing-field-guide/blob/main/pages/about/references.md" class="btn btn-sm hover-primary text-primary"><i class="fa-solid fa-pencil-alt"></i></a>
    <a role="button" data-bs-toggle="tooltip" title="Report an issue" href="https://github.com/mshadbolt/human-omics-data-sharing-field-guide/issues/new?title=Issue+on+page%3A+References&amp;body=I+would+like+to+report+an+issue+on+the+References+page+at+%60%2Freferences%60.+Description+of+the+issue%3A" class="btn btn-sm hover-primary text-primary"><i class="fa-solid fa-exclamation"></i></a>
</div>

    </h1>
    <p>Full set of references used in this website.</p>

<ol class="bibliography"><li><span id="oliveira_portuguese_2022">Oliveira, J. S., Silva, F. M. da, Oliveira, J. L., Silva, M., &amp; Melo, A. M. P. (2022). <i>The Portuguese Local European  Genome-Phenome Archive (EGA)</i>. https://doi.org/10.5281/zenodo.7186542</span>


<a href="https://zenodo.org/record/7186542" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="grimes_pathling_2022">Grimes, J., Szul, P., Metke-Jimenez, A., Lawley, M., &amp; Loi, K. (2022). Pathling: analytics on FHIR. <i>Journal of Biomedical Semantics</i>, <i>13</i>(1), 23. https://doi.org/10.1186/s13326-022-00277-1</span>


<a href="https://doi.org/10.1186/s13326-022-00277-1" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="rueda_beacon_2022">Rueda, M., Ariosa, R., Moldes, M., &amp; Rambla, J. (2022). Beacon V2 Reference Implementation: a Toolkit to enable federated sharing of genomic and phenotypic data. <i>Bioinformatics</i>, btac568. https://doi.org/10.1093/bioinformatics/btac568</span>


<a href="https://doi.org/10.1093/bioinformatics/btac568" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="duda_hl7_2022">Duda, S. N., Kennedy, N., Conway, D., Cheng, A. C., Nguyen, V., Zayas-Cab√°n, T., &amp; Harris, P. A. (2022). HL7 FHIR-based tools and initiatives to support clinical research: a scoping review. <i>Journal of the American Medical Informatics Association</i>, <i>29</i>(9), 1642‚Äì1653. https://doi.org/10.1093/jamia/ocac105</span>


<a href="https://doi.org/10.1093/jamia/ocac105" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_data_2022-1">GA4GH. (2022). <i>Data Repository Service (DRS) API</i>. Global Alliance for Genomics and Health. https://github.com/ga4gh/data-repository-service-schemas</span>


<a href="https://github.com/ga4gh/data-repository-service-schemas" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="vorisek_fast_2022">Vorisek, C. N., Lehne, M., Klopfenstein, S. A. I., Mayer, P. J., Bartschke, A., Haese, T., &amp; Thun, S. (2022). Fast Healthcare Interoperability Resources (FHIR) for Interoperability in Health Research: Systematic Review. <i>JMIR Medical Informatics</i>, <i>10</i>(7), e35724. https://doi.org/10.2196/35724</span>


<a href="https://medinform.jmir.org/2022/7/e35724" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="brandt_characterizing_2022">Brandt, P. S., Kho, A., Luo, Y., Pacheco, J. A., Walunas, T. L., Hakonarson, H., Hripcsak, G., Liu, C., Shang, N., Weng, C., Walton, N., Carrell, D. S., Crane, P. K., Larson, E., Chute, C. G., Kullo, I., Carroll, R., Denny, J., Ramirez, A., ‚Ä¶ Rasmussen, L. V. (2022). <i>Characterizing Variability of EHR-Driven Phenotype Definitions</i>. medRxiv. https://doi.org/10.1101/2022.07.10.22277390</span>


<a href="https://www.medrxiv.org/content/10.1101/2022.07.10.22277390v1" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="hartley_bioimage_2022">Hartley, M., Kleywegt, G. J., Patwardhan, A., Sarkans, U., Swedlow, J. R., &amp; Brazma, A. (2022). The BioImage Archive ‚Äì Building a Home for Life-Sciences Microscopy Data. <i>Journal of Molecular Biology</i>, <i>434</i>(11), 167505. https://doi.org/10.1016/j.jmb.2022.167505</span>


<a href="https://www.sciencedirect.com/science/article/pii/S0022283622000791" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="jacobsen_ga4gh_2022">Jacobsen, J. O. B., Baudis, M., Baynam, G. S., Beckmann, J. S., Beltran, S., Buske, O. J., Callahan, T. J., Chute, C. G., Courtot, M., Danis, D., Elemento, O., Essenwanger, A., Freimuth, R. R., Gargano, M. A., Groza, T., Hamosh, A., Harris, N. L., Kaliyaperumal, R., Lloyd, K. C. K., ‚Ä¶ Robinson, P. N. (2022). The GA4GH Phenopacket schema defines a computable representation of clinical data. <i>Nature Biotechnology</i>, <i>40</i>(6), 817‚Äì820. https://doi.org/10.1038/s41587-022-01357-4</span>


<a href="https://www.nature.com/articles/s41587-022-01357-4" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="gruendner_architecture_2022">Gruendner, J., Deppenwiese, N., Folz, M., K√∂hler, T., Kroll, B., Prokosch, H.-U., Rosenau, L., R√ºhle, M., Scheidl, M.-A., Sch√ºttler, C., Sedlmayr, B., Twrdik, A., Kiel, A., &amp; Majeed, R. W. (2022). The Architecture of a Feasibility Query Portal for Distributed COVID-19 Fast Healthcare Interoperability Resources (FHIR) Patient Data Repositories: Design and Implementation Study. <i>JMIR Medical Informatics</i>, <i>10</i>(5), e36709. https://doi.org/10.2196/36709</span>


<a href="https://europepmc.org/articles/PMC9135115" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="hernandez_applying_2022">Hernandez, S., Fairchild, K., Pemberton, M., Dahmer, J., Zhang, W., Palchuk, M. B., &amp; Topaloglu, U. (2022). Applying FHIR Genomics for Research ‚Äì From Sequencing to Database. <i>AMIA Annual Symposium Proceedings</i>, <i>2022</i>, 236‚Äì243. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9285172/</span>


<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9285172/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="horgan_b1mg_2022">Horgan, D., Boccia, S., Becker, R., Scollen, S., Merchant, A., van El, C., Hoxaj, I., Sassano, M., Pezzullo, A., &amp; Julkowska, D. (2022). <i>B1MG D1.5 Stakeholders trust in genomic data sharing landscape analysis</i>. https://doi.org/10.5281/zenodo.6382431</span>


<a href="https://zenodo.org/record/6382431" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="freeberg_european_2022">Freeberg, M. A., Fromont, L. A., D‚ÄôAltri, T., Romero, A. F., Ciges, J. I., Jene, A., Kerry, G., Moldes, M., Ariosa, R., Bahena, S., Barrowdale, D., Barbero, M. C., Fernandez-Orth, D., Garcia-Linares, C., Garcia-Rios, E., Haziza, F., Juhasz, B., Llobet, O. M., Milla, G., ‚Ä¶ Rambla, J. (2022). The European Genome-phenome Archive in 2021. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D980‚ÄìD987. https://doi.org/10.1093/nar/gkab1059</span>


<a href="http://dx.doi.org/10.1093/nar/gkab1059" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="schatz_inverting_2022">Schatz, M. C., Philippakis, A. A., Afgan, E., Banks, E., Carey, V. J., Carroll, R. J., Culotti, A., Ellrott, K., Goecks, J., Grossman, R. L., Hall, I. M., Hansen, K. D., Lawson, J., Leek, J. T., Luria, A. O. D., Mosher, S., Morgan, M., Nekrutenko, A., O‚ÄôConnor, B. D., ‚Ä¶ Wuichet, K. (2022). Inverting the model of genomics data sharing with the NHGRI Genomic Data Science Analysis, Visualization, and Informatics Lab-space. <i>Cell Genomics</i>, <i>2</i>(1). https://doi.org/10.1016/j.xgen.2021.100085</span>


<a href="https://linkinghub.elsevier.com/retrieve/pii/S2666979X21001063" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cummins_european_2022">Cummins, C., Ahamed, A., Aslam, R., Burgin, J., Devraj, R., Edbali, O., Gupta, D., Harrison, P. W., Haseeb, M., Holt, S., Ibrahim, T., Ivanov, E., Jayathilaka, S., Kadhirvelu, V., Kay, S., Kumar, M., Lathi, A., Leinonen, R., Madeira, F., ‚Ä¶ Cochrane, G. (2022). The European Nucleotide Archive in 2021. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D106‚ÄìD110. https://doi.org/10.1093/nar/gkab1051</span>


<a href="https://doi.org/10.1093/nar/gkab1051" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="moreno_expression_2022">Moreno, P., Fexova, S., George, N., Manning, J. R., Miao, Z., Mohammed, S., Mu√±oz-Pomer, A., Fullgrabe, A., Bi, Y., Bush, N., Iqbal, H., Kumbham, U., Solovyev, A., Zhao, L., Prakash, A., Garc√≠a-Seisdedos, D., Kundu, D. J., Wang, S., Walzer, M., ‚Ä¶ Papatheodorou, I. (2022). Expression Atlas update: gene and protein expression in multiple species. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D129‚ÄìD140. https://doi.org/10.1093/nar/gkab1030</span>


<a href="https://doi.org/10.1093/nar/gkab1030" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cezard_european_2022">Cezard, T., Cunningham, F., Hunt, S. E., Koylass, B., Kumar, N., Saunders, G., Shen, A., Silva, A. F., Tsukanov, K., Venkataraman, S., Flicek, P., Parkinson, H., &amp; Keane, T. M. (2022). The European Variation Archive: a FAIR resource of genomic variation for all species. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D1216‚ÄìD1220. https://doi.org/10.1093/nar/gkab960</span>


<a href="https://doi.org/10.1093/nar/gkab960" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="katz_sequence_2022">Katz, K., Shutov, O., Lapoint, R., Kimelman, M., Brister, J. R., &amp; O‚ÄôSullivan, C. (2022). The Sequence Read Archive: a decade more of explosive growth. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D387‚ÄìD390. https://doi.org/10.1093/nar/gkab1053</span>


<a href="https://doi.org/10.1093/nar/gkab1053" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="sayers_genbank_2022">Sayers, E. W., Cavanaugh, M., Clark, K., Pruitt, K. D., Schoch, C. L., Sherry, S. T., &amp; Karsch-Mizrachi, I. (2022). GenBank. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D161‚ÄìD164. https://doi.org/10.1093/nar/gkab1135</span>


<a href="https://doi.org/10.1093/nar/gkab1135" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="okido_dna_2022">Okido, T., Kodama, Y., Mashima, J., Kosuge, T., Fujisawa, T., &amp; Ogasawara, O. (2022). DNA Data Bank of Japan (DDBJ) update report 2021. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D102‚ÄìD105. https://doi.org/10.1093/nar/gkab995</span>


<a href="https://doi.org/10.1093/nar/gkab995" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cncb-ngdc_members_and_partners_database_2022">CNCB-NGDC Members and Partners. (2022). Database Resources of the National Genomics Data Center, China National Center for Bioinformation in 2022. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D27‚ÄìD38. https://doi.org/10.1093/nar/gkab951</span>


<a href="https://doi.org/10.1093/nar/gkab951" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="perez-riverol_pride_2022">Perez-Riverol, Y., Bai, J., Bandla, C., Garc√≠a-Seisdedos, D., Hewapathirana, S., Kamatchinathan, S., Kundu, D. J., Prakash, A., Frericks-Zipper, A., Eisenacher, M., Walzer, M., Wang, S., Brazma, A., &amp; Vizca√≠no, J. A. (2022). The PRIDE database resources in 2022: a hub for mass spectrometry-based proteomics evidences. <i>Nucleic Acids Research</i>, <i>50</i>(D1), D543‚ÄìD552. https://doi.org/10.1093/nar/gkab1038</span>


<a href="https://doi.org/10.1093/nar/gkab1038" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="brambilla_pisoni_apropos_2022">Brambilla Pisoni, G., &amp; Taddeo, M. (2022). Apropos Data Sharing: Abandon the Distrust and Embrace the Opportunity. <i>DNA and Cell Biology</i>, <i>41</i>(1), 11‚Äì15. https://doi.org/10.1089/dna.2021.0501</span>


<a href="https://www.liebertpub.com/doi/10.1089/dna.2021.0501" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="chatterjee_hl7_2022">Chatterjee, A., Pahari, N., &amp; Prinz, A. (2022). HL7 FHIR with SNOMED-CT to Achieve Semantic and Structural Interoperability in Personal Health Data: A Proof-of-Concept Study. <i>Sensors</i>, <i>22</i>(10), 3756. https://doi.org/10.3390/s22103756</span>


<a href="https://www.mdpi.com/1424-8220/22/10/3756" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_ga4gh_2022">GA4GH. (2022). <i>GA4GH Passports and the Authorization and Authentication Infrastructure</i>. https://www.ga4gh.org/news/ga4gh-passports-and-the-authorization-and-authentication-infrastructure/</span>


<a href="https://www.ga4gh.org/news/ga4gh-passports-and-the-authorization-and-authentication-infrastructure/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cncb-ngdc_database_2022">CNCB-NGDC. (2022). <i>Database Resources of the National Genomics Data Center, China National Center for Bioinformation in 2022. - Abstract - Europe PMC</i>. https://europepmc.org/article/MED/34718731</span>


<a href="https://europepmc.org/article/MED/34718731" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="nanyang_technological_university_libguides_2022">University, N. T. (2022). <i>LibGuides: Research Data Management: Sensitive Data</i>. https://libguides.ntu.edu.sg/rdm/sensitivedata</span>


<a href="https://libguides.ntu.edu.sg/rdm/sensitivedata" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ena_ena_2022">ENA. (2022). <i>ENA: Guidelines and Tutorials ‚Äî ENA Training Modules 1 documentation</i>. https://ena-docs.readthedocs.io/en/latest/#</span>


<a href="https://ena-docs.readthedocs.io/en/latest/#" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ega_prepare_2022">EGA. (2022). <i>Prepare XMLs - EGA European Genome-Phenome Archive</i>. https://ega-archive.org/submission/sequence/programmatic_submissions/prepare_xml</span>


<a href="https://ega-archive.org/submission/sequence/programmatic_submissions/prepare_xml" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ladewig_ga4gh_2022">Ladewig, M. S., Jacobsen, J. O. B., Wagner, A. H., Danis, D., El Kassaby, B., Gargano, M., Groza, T., Baudis, M., Steinhaus, R., Seelow, D., Bechrakis, N. E., Mungall, C. J., Schofield, P. N., Elemento, O., Smith, L., McMurry, J. A., Munoz-Torres, M., Haendel, M. A., &amp; Robinson, P. N. (2022). GA4GH Phenopackets: A Practical Introduction. <i>Advanced Genetics</i>, <i>n/a</i>(n/a), 2200016. https://doi.org/10.1002/ggn2.202200016</span>


<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/ggn2.202200016" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="nature_data_2022">nature. (2022). <i>Data Repository Guidance \textbar Scientific Data</i>. https://www.nature.com/sdata/policies/repositories</span>


<a href="https://www.nature.com/sdata/policies/repositories" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="springer_nature_mandated_2022">Nature, S. (2022). <i>Mandated data types \textbar Authors \textbar Springer Nature</i>. https://www.springernature.com/gp/authors/research-data-policy/repositories-socsci/19540364</span>


<a href="https://www.springernature.com/gp/authors/research-data-policy/repositories-socsci/19540364" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="rambla_beacon_2022">Rambla, J., Baudis, M., Ariosa, R., Beck, T., Fromont, L. A., Navarro, A., Paloots, R., Rueda, M., Saunders, G., Singh, B., Spalding, J. D., T√∂rnroos, J., Vasallo, C., Veal, C. D., &amp; Brookes, A. J. (2022). Beacon v2 and Beacon networks: A ‚Äúlingua franca‚Äù for federated data discovery in biomedical genomics, and beyond. <i>Human Mutation</i>, <i>43</i>(6), 791‚Äì799. https://doi.org/10.1002/humu.24369</span>


<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/humu.24369" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="fairsharing_team_fairsharing_2022">team, F. A. I. R. (2022). <i>FAIRsharing \textbar Home</i>. https://fairsharing.org/</span>


<a href="https://fairsharing.org/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="nielsen_returning_2022">Nielsen, J. L., Johnston, C., O‚ÄôBrien, T., &amp; Tyrrell, V. J. (2022). Returning raw genomic data: rights of research participants and obligations of health care professionals. <i>Medical Journal of Australia</i>, <i>216</i>(11), 550‚Äì552. https://doi.org/10.5694/mja2.51546</span>


<a href="https://onlinelibrary.wiley.com/doi/abs/10.5694/mja2.51546" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="kauffmann_arrayexpress_2022">Kauffmann, A., Emam, I., &amp; Schubert, M. (2022). <i>ArrayExpress: Access the ArrayExpress Microarray Database at EBI and build Bioconductor data structures: ExpressionSet, AffyBatch, NChannelSet</i>. Bioconductor version: Release (3.15). https://doi.org/10.18129/B9.bioc.ArrayExpress</span>


<a href="https://bioconductor.org/packages/ArrayExpress/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="davis_geoquery_2022">Davis, S. (2022). <i>GEOquery: Get data from NCBI Gene Expression Omnibus (GEO)</i>. Bioconductor version: Release (3.15). https://doi.org/10.18129/B9.bioc.GEOquery</span>


<a href="https://bioconductor.org/packages/GEOquery/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_data_2022">GA4GH. (2022). <i>Data Repository Service</i>. https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/</span>


<a href="https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_drs_2022">GA4GH. (2022). <i>DRS API: Enabling Cloud-Based Data Access and Retrieval</i>. https://www.ga4gh.org/news/drs-api-enabling-cloud-based-data-access-and-retrieval/</span>


<a href="https://www.ga4gh.org/news/drs-api-enabling-cloud-based-data-access-and-retrieval/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_welcome_2022">GA4GH. (2022). <i>Welcome to the technical documentation for the GA4GH Pedigree Standard! ‚Äî 0.1 documentation</i>. https://pedigree.readthedocs.io/en/latest/</span>


<a href="https://pedigree.readthedocs.io/en/latest/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_ga4gh_2022-1">GA4GH. (2022). <i>GA4GH WES API enables portable genomic analysis</i>. https://www.ga4gh.org/news/ga4gh-wes-api-enables-portable-genomic-analysis/</span>


<a href="https://www.ga4gh.org/news/ga4gh-wes-api-enables-portable-genomic-analysis/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_workflow_2022">GA4GH. (2022). <i>Workflow Execution Service</i>. https://ga4gh.github.io/workflow-execution-service-schemas/docs/</span>


<a href="https://ga4gh.github.io/workflow-execution-service-schemas/docs/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_ga4gh_2022-2">GA4GH. (2022). <i>GA4GH DRS Client Documentation ‚Äî GA4GH DRS Client documentation</i>. https://ga4gh-drs-client.readthedocs.io/en/latest/</span>


<a href="https://ga4gh-drs-client.readthedocs.io/en/latest/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="martinez-garcia_fairness_2022">Mart√≠nez-Garc√≠a, A., Cangioli, G., Chronaki, C., L√∂be, M., Beyan, O., Juehne, A., &amp; Parra-Calder√≥n, C. L. (2022). FAIRness for FHIR: Towards Making Health Datasets FAIR Using HL7 FHIR. <i>MEDINFO 2021: One World, One Health ‚Äì Global Partnership for Digital Innovation</i>, 22‚Äì26. https://doi.org/10.3233/SHTI220024</span>


<a href="https://ebooks.iospress.nl/doi/10.3233/SHTI220024" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="fhir_summary_2022">FHIR. (2022). <i>Summary - FHIR v4.3.0</i>. https://www.hl7.org/fhir/summary.html</span>


<a href="https://www.hl7.org/fhir/summary.html" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ohdsi_omop_2022">OHDSI. (2022). <i>OMOP Common Data Model ‚Äì OHDSI</i>. https://www.ohdsi.org/data-standardization/the-common-data-model/</span>


<a href="https://www.ohdsi.org/data-standardization/the-common-data-model/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="anagnostou_emerging_2021">Anagnostou, P., Capocasa, M., Brisighelli, F., Battaggia, C., &amp; Destro Bisol, G. (2021). The emerging complexity of Open Science: assessing Intelligent Data Openness in Genomic Anthropology and Human Genomics. <i>Journal of Anthropological Sciences = Rivista Di Antropologia : JASS / Istituto Italiano Di Antropologia</i>, <i>99</i>. https://doi.org/10.4436/JASS.99016</span>


<a href="http://dx.doi.org/10.4436/JASS.99016" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cabili_empirical_2021">Cabili, M. N., Lawson, J., Saltzman, A., Rushton, G., O‚ÄôRourke, P., Wilbanks, J., Rodriguez, L. L., Nyronen, T., Courtot, M., Donnelly, S., &amp; Philippakis, A. A. (2021). Empirical validation of an automated approach to data use oversight. <i>Cell Genomics</i>, <i>1</i>(2), 100031. https://doi.org/10.1016/j.xgen.2021.100031</span>


<a href="https://linkinghub.elsevier.com/retrieve/pii/S2666979X21000380" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="rehm_ga4gh_2021">Rehm, H. L., Page, A. J. H., Smith, L., Adams, J. B., Alterovitz, G., Babb, L. J., Barkley, M. P., Baudis, M., Beauvais, M. J. S., Beck, T., Beckmann, J. S., Beltran, S., Bernick, D., Bernier, A., Bonfield, J. K., Boughtwood, T. F., Bourque, G., Bowers, S. R., Brookes, A. J., ‚Ä¶ Birney, E. (2021). GA4GH: International policies and standards for data sharing across genomic research and healthcare. <i>Cell Genomics</i>, <i>1</i>(2), 100029. https://doi.org/10.1016/j.xgen.2021.100029</span>


<a href="https://www.sciencedirect.com/science/article/pii/S2666979X21000367" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="wagner_ga4gh_2021">Wagner, A. H., Babb, L., Alterovitz, G., Baudis, M., Brush, M., Cameron, D. L., Cline, M., Griffith, M., Griffith, O. L., Hunt, S. E., Kreda, D., Lee, J. M., Li, S., Lopez, J., Moyer, E., Nelson, T., Patel, R. Y., Riehle, K., Robinson, P. N., ‚Ä¶ Hart, R. K. (2021). The GA4GH Variation Representation Specification: A computational framework for variation representation and federated identification. <i>Cell Genomics</i>, <i>1</i>(2), 100027. https://doi.org/10.1016/j.xgen.2021.100027</span>


<a href="https://www.sciencedirect.com/science/article/pii/S2666979X21000343" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="voisin_ga4gh_2021">Voisin, C., Linden, M., Dyke, S. O. M., Bowers, S. R., Alper, P., Barkley, M. P., Bernick, D., Chao, J., Courtot, M., Jeanson, F., Konopko, M. A., Kuba, M., Lawson, J., Leinonen, J., Li, S., Ota Wang, V., Philippakis, A. A., Reinold, K., Rushton, G. A., ‚Ä¶ Nyr√∂nen, T. H. (2021). GA4GH Passport standard for digital identity and access permissions. <i>Cell Genomics</i>, <i>1</i>(2), 100030. https://doi.org/10.1016/j.xgen.2021.100030</span>


<a href="https://www.sciencedirect.com/science/article/pii/S2666979X21000379" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="lawson_data_2021">Lawson, J., Cabili, M. N., Kerry, G., Boughtwood, T., Thorogood, A., Alper, P., Bowers, S. R., Boyles, R. R., Brookes, A. J., Brush, M., Burdett, T., Clissold, H., Donnelly, S., Dyke, S. O. M., Freeberg, M. A., Haendel, M. A., Hata, C., Holub, P., Jeanson, F., ‚Ä¶ Courtot, M. (2021). The Data Use Ontology to streamline responsible access to human biomedical datasets. <i>Cell Genomics</i>, <i>1</i>(2), 100028. https://doi.org/10.1016/j.xgen.2021.100028</span>


<a href="https://www.sciencedirect.com/science/article/pii/S2666979X21000355" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="hekel_privacy-preserving_2021">Hekel, R., Budis, J., Kucharik, M., Radvanszky, J., P√∂s, Z., &amp; Szemes, T. (2021). Privacy-preserving storage of sequenced genomic data. <i>BMC Genomics</i>, <i>22</i>(1), 712. https://doi.org/10.1186/s12864-021-07996-2</span>


<a href="http://dx.doi.org/10.1186/s12864-021-07996-2" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="chen_genome_2021">Chen, T., Chen, X., Zhang, S., Zhu, J., Tang, B., Wang, A., Dong, L., Zhang, Z., Yu, C., Sun, Y., Chi, L., Chen, H., Zhai, S., Sun, Y., Lan, L., Zhang, X., Xiao, J., Bao, Y., Wang, Y., ‚Ä¶ Zhao, W. (2021). The genome sequence archive family: toward explosive data growth and diverse data types. <i>Genomics, Proteomics &amp; Bioinformatics / Beijing Genomics Institute</i>. https://doi.org/10.1016/j.gpb.2021.08.001</span>


<a href="http://dx.doi.org/10.1016/j.gpb.2021.08.001" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="oestreich_privacy_2021">Oestreich, M., Chen, D., Schultze, J. L., Fritz, M., &amp; Becker, M. (2021). Privacy considerations for sharing genomics data. <i>EXCLI Journal</i>, <i>20</i>, 1243‚Äì1260. https://doi.org/10.17179/excli2021-4002</span>


<a href="https://www.excli.de/index.php/excli/article/view/4002" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="tedersoo_data_2021">Tedersoo, L., K√ºngas, R., Oras, E., K√∂ster, K., Eenmaa, H., Leijen, √Ñ., Pedaste, M., Raju, M., Astapova, A., Lukner, H., Kogermann, K., &amp; Sepp, T. (2021). Data sharing practices and data availability upon request differ across scientific disciplines. <i>Scientific Data</i>, <i>8</i>(1), 192. https://doi.org/10.1038/s41597-021-00981-0</span>


<a href="http://www.nature.com/articles/s41597-021-00981-0" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="carroll_operationalizing_2021">Carroll, S. R., Herczog, E., Hudson, M., Russell, K., &amp; Stall, S. (2021). Operationalizing the CARE and FAIR Principles for Indigenous data futures. <i>Scientific Data</i>, <i>8</i>(1), 108. https://doi.org/10.1038/s41597-021-00892-0</span>


<a href="http://dx.doi.org/10.1038/s41597-021-00892-0" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="vernon_metabolic_2021">Vernon, S. T., Tang, O., Kim, T., Chan, A. S., Kott, K. A., Park, J., Hansen, T., Koay, Y. C., Grieve, S. M., O‚ÄôSullivan, J. F., Yang, J. Y., &amp; Figtree, G. A. (2021). Metabolic Signatures in Coronary Artery Disease: Results from the BioHEART-CT Study. <i>Cells</i>, <i>10</i>(5). https://doi.org/10.3390/cells10050980</span>


<a href="http://dx.doi.org/10.3390/cells10050980" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="haas_ctrl_2021">Haas, M. A., Teare, H., Prictor, M., Ceregra, G., Vidgen, M. E., Bunker, D., Kaye, J., &amp; Boughtwood, T. (2021). ‚ÄôCTRL‚Äô: an online, Dynamic Consent and participant engagement platform working towards solving the complexities of consent in genomic research. <i>European Journal of Human Genetics</i>, <i>29</i>(4), 687‚Äì698. https://doi.org/10.1038/s41431-020-00782-w</span>


<a href="http://dx.doi.org/10.1038/s41431-020-00782-w" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cannon_repository_2021">Cannon, M., Graf, C., McNeice, K., Chan, W. M., Callaghan, S., Carnevale, I., Cranston, I., Edmunds, S. C., Everitt, N., Ganley, E., Hrynaszkiewicz, I., Khodiyar, V. K., Leary, A., Lemberger, T., MacCallum, C. J., Murray, H., Sharples, K., Soares E Silva, M., Wright, G., ‚Ä¶ (Moderator) Sansone, S.-A. (2021). <i>Repository Features to Help Researchers: An invitation to a dialogue</i>. Zenodo. https://doi.org/10.5281/zenodo.4683794</span>


<a href="https://zenodo.org/record/4683794" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="dursi_candig_2021">Dursi, L. J., Bozoky, Z., de Borja, R., Li, J., Bujold, D., Lipski, A., Rashid, S. F., Sethi, A., Memon, N., Naidoo, D., Coral-Sasso, F., Wong, M., Quirion, P.-O., Lu, Z., Agarwal, S., Pavlov, K., Ponomarev, A., Husic, M., Pace, K., ‚Ä¶ Brudno, M. (2021). Candig: secure federated genomic queries and analyses across jurisdictions. <i>BioRxiv</i>. https://doi.org/10.1101/2021.03.30.434101</span>


<a href="http://biorxiv.org/lookup/doi/10.1101/2021.03.30.434101" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="dove_raising_2021">Dove, E. S., Chen, J., &amp; Loideain, N. N. (2021). Raising standards for global data-sharing. <i>Science</i>, <i>371</i>(6525), 133‚Äì134. https://doi.org/10.1126/science.abf4286</span>


<a href="https://www.science.org/doi/10.1126/science.abf4286" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="mukherjee_genomes_2021">Mukherjee, S., Stamatis, D., Bertsch, J., Ovchinnikova, G., Sundaramurthi, J. C., Lee, J., Kandimalla, M., Chen, I.-M. A., Kyrpides, N. C., &amp; Reddy, T. B. K. (2021). Genomes OnLine Database (GOLD) v.8: overview and updates. <i>Nucleic Acids Research</i>, <i>49</i>(D1), D723‚ÄìD733. https://doi.org/10.1093/nar/gkaa983</span>


<a href="https://europepmc.org/articles/PMC7778979" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="observational_health_data_sciences_and_informatics_chapter_2021">Informatics, O. H. D. S. and. (2021). <i>Chapter 4 The Common Data Model \textbar The Book of OHDSI</i>. https://ohdsi.github.io/TheBookOfOhdsi/</span>


<a href="https://ohdsi.github.io/TheBookOfOhdsi/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="tanjo_practical_2021">Tanjo, T., Kawai, Y., Tokunaga, K., Ogasawara, O., &amp; Nagasaki, M. (2021). Practical guide for managing large-scale human genome data in research. <i>Journal of Human Genetics</i>, <i>66</i>(1), 39‚Äì52. https://doi.org/10.1038/s10038-020-00862-1</span>


<a href="http://www.nature.com/articles/s10038-020-00862-1" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="odoherty_toward_2021">O‚ÄôDoherty, K. C., Shabani, M., Dove, E. S., Bentzen, H. B., Borry, P., Burgess, M. M., Chalmers, D., De Vries, J., Eckstein, L., Fullerton, S. M., Juengst, E., Kato, K., Kaye, J., Knoppers, B. M., Koenig, B. A., Manson, S. M., McGrail, K. M., McGuire, A. L., Meslin, E. M., ‚Ä¶ Burke, W. (2021). Toward better governance of human genomic data. <i>Nature Genetics</i>, <i>53</i>(1), 2‚Äì8. https://doi.org/10.1038/s41588-020-00742-6</span>


<a href="http://dx.doi.org/10.1038/s41588-020-00742-6" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="jin_cineca_2021">Jin, V., Cook, J., Dubeau, A., Dai, R., Freitas, T., Dursi, J., Gill, E., Winsor, G., Courtot, M., Subbarao, P., &amp; Brinkman, F. (2021). CINECA synthetic cohort NA Canada CHILD [CC-BY-NC-SA]. <i>Zenodo</i>. https://doi.org/10.5281/zenodo.5122832</span>


<a href="https://zenodo.org/record/5122832" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cineca_uncovering_2021">CINECA. (2021). <i>Uncovering metadata from semi-structured cohort data ‚Äî CINECA - Common Infrastructure for National Cohorts in Europe, Canada, and Africa</i>. https://www.cineca-project.eu/blog-all/uncovering-metadata-from-semi-structured-cohort-data</span>


<a href="https://www.cineca-project.eu/blog-all/uncovering-metadata-from-semi-structured-cohort-data" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cineca_lexmapr_2021">CINECA. (2021). <i>LexMapr - A rule-based text-mining tool for ontology term mapping and classification ‚Äî CINECA - Common Infrastructure for National Cohorts in Europe, Canada, and Africa</i>. https://www.cineca-project.eu/blog-all/lexmapr-a-rule-based-text-mining-tool-for-ontology-term-mapping-and-classification</span>


<a href="https://www.cineca-project.eu/blog-all/lexmapr-a-rule-based-text-mining-tool-for-ontology-term-mapping-and-classification" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cineca_assigning_2021">CINECA. (2021). <i>Assigning standard descriptors to free text ‚Äî CINECA - Common Infrastructure for National Cohorts in Europe, Canada, and Africa</i>. https://www.cineca-project.eu/blog-all/assigning-standard-descriptors-to-free-text</span>


<a href="https://www.cineca-project.eu/blog-all/assigning-standard-descriptors-to-free-text" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_github_2021">GA4GH. (2021). <i>GitHub - EBISPOT/DUO: Ontology for consent codes and data use requirements</i>. https://github.com/EBISPOT/DUO</span>


<a href="https://github.com/EBISPOT/DUO" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ga4gh_data-securityaaiconnectprofilemd_2021">GA4GH. (2021). <i>data-security/AAIConnectProfile.md at master ¬∑ ga4gh/data-security ¬∑ GitHub</i>. https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#abstract</span>


<a href="https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#abstract" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="embl-ebi_embl-ebi_2021">EMBL-EBI. (2021). <i>EMBL-EBI Impact report 2021 ‚Äì EMBL Documents</i>. https://www.embl.org/documents/document/embl-ebi-impact-report-2021/</span>


<a href="https://www.embl.org/documents/document/embl-ebi-impact-report-2021/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cellpress_authors_2021">CellPress. (2021). <i>Author‚Äôs guide: Standardized datatypes, datatype specific repositories, and general-purpose repositories recommended by Cell Press</i>. https://www.cell.com/pb-assets/journals/research/cellpress/data/RecommendRepositories-1621989644133.pdf</span>


<a href="https://www.cell.com/pb-assets/journals/research/cellpress/data/RecommendRepositories-1621989644133.pdf" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="ingena_genomic_2021">InGeNA. (2021). <i>Genomic Data in Australia: An industry perspective on clarity, certainty and standardisation</i>.</span>


</li>
<li><span id="bull_ethics_2020">Bull, S., &amp; Bhagwandin, N. (2020). The ethics of data sharing and biobanking in health research. <i>Wellcome Open Research</i>, <i>5</i>, 270. https://doi.org/10.12688/wellcomeopenres.16351.1</span>


<a href="http://dx.doi.org/10.12688/wellcomeopenres.16351.1" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="arita_international_2020">Arita, M., Karsch-Mizrachi, I., &amp; Cochrane, G. (2020). The international nucleotide sequence database collaboration. <i>Nucleic Acids Research</i>, <i>49</i>(D1), D121‚ÄìD124. https://doi.org/10.1093/nar/gkaa967</span>


<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7778961/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="vidgen_sharing_2020">Vidgen, M. E., Kaladharan, S., Malacova, E., Hurst, C., &amp; Waddell, N. (2020). Sharing genomic data from clinical testing with researchers: public survey of expectations of clinical genomic data management in Queensland, Australia. <i>BMC Medical Ethics</i>, <i>21</i>(1), 119. https://doi.org/10.1186/s12910-020-00563-6</span>


<a href="https://doi.org/10.1186/s12910-020-00563-6" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="byrd_responsible_2020">Byrd, J. B., Greene, A. C., Prasad, D. V., Jiang, X., &amp; Greene, C. S. (2020). Responsible, practical genomic data sharing that accelerates research. <i>Nature Reviews. Genetics</i>, <i>21</i>(10), 615‚Äì629. https://doi.org/10.1038/s41576-020-0257-5</span>


<a href="http://www.nature.com/articles/s41576-020-0257-5" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="osterman_improving_2020">Osterman, T. J., Terry, M., &amp; Miller, R. S. (2020). Improving cancer data interoperability: the promise of the minimal common oncology data elements (mcode) initiative. <i>JCO Clinical Cancer Informatics</i>, <i>4</i>, 993‚Äì1001. https://doi.org/10.1200/CCI.20.00059</span>


<a href="http://dx.doi.org/10.1200/CCI.20.00059" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="queensland_genomics_blueprint_2020">Queensland Genomics. (2020). <i>Blueprint for a National Approach to Genomic Information Management (NAGIM)</i>. Queensland Health. https://queenslandgenomics.org/capability-initiatives/national-approach-to-genomics-information-management/</span>


<a href="https://queenslandgenomics.org/capability-initiatives/national-approach-to-genomics-information-management/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="bovenberg_how_2020">Bovenberg, J., Peloquin, D., Bierer, B., Barnes, M., &amp; Knoppers, B. M. (2020). How to fix the GDPR‚Äôs frustration of global biomedical research. <i>Science</i>, <i>370</i>(6512), 40‚Äì42. https://doi.org/10.1126/science.abd2499</span>


<a href="https://www.science.org/doi/10.1126/science.abd2499" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="dive_public_2020">Dive, L., Critchley, C., Otlowski, M., Mason, P., Wiersma, M., Light, E., Stewart, C., Kerridge, I., &amp; Lipworth, W. (2020). Public trust and global biobank networks. <i>BMC Medical Ethics</i>, <i>21</i>(1), 73. https://doi.org/10.1186/s12910-020-00515-0</span>


<a href="http://dx.doi.org/10.1186/s12910-020-00515-0" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="jowett_genomic_2020">Jowett, S., Dallaston, E., &amp; Bennett, B. (2020). Genomic Research and Data-Sharing : Time to Revisit Australian Laws? <i>The University of Queensland Law Journal</i>, <i>39</i>(2), 341‚Äì369. https://eprints.qut.edu.au/203714/</span>


<a href="https://eprints.qut.edu.au/203714/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="hudson_rights_2020">Hudson, M., Garrison, N. A., Sterling, R., Caron, N. R., Fox, K., Yracheta, J., Anderson, J., Wilcox, P., Arbour, L., Brown, A., Taualii, M., Kukutai, T., Haring, R., Te Aika, B., Baynam, G. S., Dearden, P. K., Chagn√©, D., Malhi, R. S., Garba, I., ‚Ä¶ Carroll, S. R. (2020). Rights, interests and expectations: Indigenous perspectives on unrestricted access to genomic data. <i>Nature Reviews Genetics</i>, <i>21</i>(6), 377‚Äì384. https://doi.org/10.1038/s41576-020-0228-x</span>


<a href="https://www.nature.com/articles/s41576-020-0228-x" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="sim_time_2020">Sim, I., Stebbins, M., Bierer, B. E., Butte, A. J., Drazen, J., Dzau, V., Hernandez, A. F., Krumholz, H. M., Lo, B., Munos, B., Perakslis, E., Rockhold, F., Ross, J. S., Terry, S. F., Yamamoto, K. R., Zarin, D. A., &amp; Li, R. (2020). Time for NIH to lead on data sharing. <i>Science</i>, <i>367</i>(6484), 1308‚Äì1309. https://doi.org/10.1126/science.aba4456</span>


<a href="https://www.science.org/doi/10.1126/science.aba4456" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="haug_metabolights_2020">Haug, K., Cochrane, K., Nainala, V. C., Williams, M., Chang, J., Jayaseelan, K. V., &amp; O‚ÄôDonovan, C. (2020). MetaboLights: a resource evolving in response to the needs of its scientific community. <i>Nucleic Acids Research</i>, <i>48</i>(D1), D440‚ÄìD444. https://doi.org/10.1093/nar/gkz1019</span>


<a href="https://doi.org/10.1093/nar/gkz1019" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="deutsch_proteomexchange_2020">Deutsch, E. W., Bandeira, N., Sharma, V., Perez-Riverol, Y., Carver, J. J., Kundu, D. J., Garc√≠a-Seisdedos, D., Jarnuczak, A. F., Hewapathirana, S., Pullman, B. S., Wertz, J., Sun, Z., Kawano, S., Okuda, S., Watanabe, Y., Hermjakob, H., MacLean, B., MacCoss, M. J., Zhu, Y., ‚Ä¶ Vizca√≠no, J. A. (2020). The ProteomeXchange consortium in 2020: enabling ‚Äòbig data‚Äô approaches in proteomics. <i>Nucleic Acids Research</i>, <i>48</i>(D1), D1145‚ÄìD1152. https://doi.org/10.1093/nar/gkz984</span>


<a href="https://doi.org/10.1093/nar/gkz984" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="alper_data_2020">Alper, P. (2020). <i>Data Protection in Biomedical Research</i>. https://doi.org/10.5281/zenodo.5078280</span>


<a href="https://zenodo.org/record/5078280" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="jin_cineca_2020">Jin, V., &amp; Brinkman, F. (2020). CINECA Cohort Level metadata Representation D3.1. <i>Zenodo</i>. https://doi.org/10.5281/zenodo.4575460</span>


<a href="https://zenodo.org/record/4575460" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="keane_ega_2020">Keane, T., Rambla, J., Spalding, D., Flicek, P., Navarro, A., &amp; Parkinson, H. (2020). <i>EGA Federation: Structure and organisation</i>.</span>


</li>
<li><span id="kerry_ega_2020">Kerry, G., Keane, T., Rambla, J., Spalding, D., Flicek, P., Navarro, A., &amp; Parkinson, H. (2020). <i>EGA Federated Node Operations</i>. https://ega-archive.org/files/EGA-Node-Operations-v1.pdf</span>


<a href="https://ega-archive.org/files/EGA-Node-Operations-v1.pdf" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="openaire_20200429_openaire_2020">OpenAIRE. (2020). <i>20200429_OpenAIRE Legal Policy Webinar for researchers - YouTube</i>. https://www.youtube.com/watch?v=85x6Rigl2-o</span>


<a href="https://www.youtube.com/watch?v=85x6Rigl2-o" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="prictor_australian_2020">Prictor, M., Huebner, S., Teare, H. J. A., Burchill, L., &amp; Kaye, J. (2020). Australian Aboriginal and Torres Strait Islander Collections of Genetic Heritage: The Legal, Ethical and Practical Considerations of a Dynamic Consent Approach to Decision Making. <i>Journal of Law, Medicine &amp; Ethics</i>, <i>48</i>(1), 205‚Äì217. https://doi.org/10.1177/1073110520917012</span>


<a href="https://www.cambridge.org/core/journals/journal-of-law-medicine-and-ethics/article/australian-aboriginal-and-torres-strait-islander-collections-of-genetic-heritage-the-legal-ethical-and-practical-considerations-of-a-dynamic-consent-approach-to-decision-making/B094693EE78CA8DBB725E01B12CCB5F0#" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="wall_genomeasia_2019">Wall, J. D., Stawiski, E. W., Ratan, A., Kim, H. L., Kim, C., Gupta, R., Suryamohan, K., Gusareva, E. S., Purbojati, R. W., Bhangale, T., Stepanov, V., Kharkov, V., Schr√∂der, M. S., Ramprasad, V., Tom, J., Durinck, S., Bei, Q., Li, J., Guillory, J., ‚Ä¶ GenomeAsia100K Consortium. (2019). The GenomeAsia 100K Project enables genetic discoveries across Asia. <i>Nature</i>, <i>576</i>(7785), 106‚Äì111. https://doi.org/10.1038/s41586-019-1793-z</span>


<a href="https://www.nature.com/articles/s41586-019-1793-z" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="stark_australian_2019">Stark, Z., Boughtwood, T., Phillips, P., Christodoulou, J., Hansen, D. P., Braithwaite, J., Newson, A. J., Gaff, C. L., Sinclair, A. H., &amp; North, K. N. (2019). Australian Genomics: A Federated Model for Integrating Genomics into Healthcare. <i>American Journal of Human Genetics</i>, <i>105</i>(1), 7‚Äì14. https://doi.org/10.1016/j.ajhg.2019.06.003</span>


<a href="http://dx.doi.org/10.1016/j.ajhg.2019.06.003" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="fernandez-orth_european_2019">Fernandez-Orth, D., Lloret-Villas, A., &amp; Rambla de Argila, J. (2019). European Genome-Phenome Archive (EGA) - Granular Solutions for the Next 10 Years. <i>2019 IEEE 32nd International Symposium on Computer-Based Medical Systems (CBMS)</i>, 4‚Äì6. https://doi.org/10.1109/CBMS.2019.00011</span>


<a href="https://ieeexplore.ieee.org/document/8787508/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="hughes_harmonization_2019">Hughes, L. R., Grossman, R. L., Flamig, Z., Prokhorenkov, A., Lukowski, M., Fitzsimons, M., Lichtenberg, T., &amp; Tang, Y. (2019). Harmonization of clinical data across Gen3 data commons. <i>JCO</i>, <i>37</i>(15_suppl), e18094‚Äìe18094. https://doi.org/10.1200/JCO.2019.37.15_suppl.e18094</span>


<a href="http://ascopubs.org/doi/10.1200/JCO.2019.37.15_suppl.e18094" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="nicol_consent_2019">Nicol, D., Eckstein, L., Bentzen, H. B., Borry, P., Burgess, M., Burke, W., Chalmers, D., Cho, M., Dove, E., Fullerton, S., Ida, R., Kato, K., Kaye, J., Koenig, B., Manson, S., McGrail, K., Meslin, E., O‚ÄôDoherty, K., Prainsack, B., ‚Ä¶ de Vries, J. (2019). Consent insufficient for data release. <i>Science</i>, <i>364</i>(6439), 445‚Äì446. https://doi.org/10.1126/science.aax0892</span>


<a href="https://www.science.org/doi/10.1126/science.aax0892" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="amann_consent_2019">Amann, R. I., Baichoo, S., Blencowe, B. J., Bork, P., Borodovsky, M., Brooksbank, C., Chain, P. S. G., Colwell, R. R., Daffonchio, D. G., Danchin, A., de Lorenzo, V., Dorrestein, P. C., Finn, R. D., Fraser, C. M., Gilbert, J. A., Hallam, S. J., Hugenholtz, P., Ioannidis, J. P. A., Jansson, J. K., ‚Ä¶ Xenarios, I. (2019). Consent insufficient for data release‚ÄîResponse. <i>Science</i>, <i>364</i>(6439), 446‚Äì446. https://doi.org/10.1126/science.aax7509</span>


<a href="https://www.science.org/doi/10.1126/science.aax7509" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="sansone_fairsharing_2019">Sansone, S.-A., McQuilton, P., Rocca-Serra, P., Gonzalez-Beltran, A., Izzo, M., Lister, A. L., &amp; Thurston, M. (2019). FAIRsharing as a community approach to standards, repositories and policies. <i>Nature Biotechnology</i>, <i>37</i>(4), 358‚Äì367. https://doi.org/10.1038/s41587-019-0080-8</span>


<a href="https://www.nature.com/articles/s41587-019-0080-8" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="kalkman_responsible_2019">Kalkman, S., Mostert, M., Gerlinger, C., van Delden, J. J. M., &amp; van Thiel, G. J. M. W. (2019). Responsible data sharing in international health research: a systematic review of principles and norms. <i>BMC Medical Ethics</i>, <i>20</i>(1), 21. https://doi.org/10.1186/s12910-019-0359-9</span>


<a href="http://dx.doi.org/10.1186/s12910-019-0359-9" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="vaske_data_2019">Vaske, O. M., &amp; Haussler, D. (2019). Data sharing for pediatric cancers. <i>Science</i>, <i>363</i>(6432), 1125‚Äì1125. https://doi.org/10.1126/science.aax2739</span>


<a href="https://www.science.org/doi/10.1126/science.aax2739" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="courtot_biosamples_2019">Courtot, M., Cherubin, L., Faulconbridge, A., Vaughan, D., Green, M., Richardson, D., Harrison, P., Whetzel, P. L., Parkinson, H., &amp; Burdett, T. (2019). BioSamples database: an updated sample metadata hub. <i>Nucleic Acids Research</i>, <i>47</i>(D1), D1172‚ÄìD1178. https://doi.org/10.1093/nar/gky1061</span>


<a href="https://doi.org/10.1093/nar/gky1061" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="amann_toward_2019">Amann, R. I., Baichoo, S., Blencowe, B. J., Bork, P., Borodovsky, M., Brooksbank, C., Chain, P. S. G., Colwell, R. R., Daffonchio, D. G., Danchin, A., de Lorenzo, V., Dorrestein, P. C., Finn, R. D., Fraser, C. M., Gilbert, J. A., Hallam, S. J., Hugenholtz, P., Ioannidis, J. P. A., Jansson, J. K., ‚Ä¶ Xenarios, I. (2019). Toward unrestricted use of public genomic data. <i>Science</i>, <i>363</i>(6425), 350‚Äì352. https://doi.org/10.1126/science.aaw1280</span>


<a href="https://www.science.org/doi/10.1126/science.aaw1280" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="athar_arrayexpress_2019">Athar, A., F√ºllgrabe, A., George, N., Iqbal, H., Huerta, L., Ali, A., Snow, C., Fonseca, N. A., Petryszak, R., Papatheodorou, I., Sarkans, U., &amp; Brazma, A. (2019). ArrayExpress update ‚Äì from bulk to single-cell expression data. <i>Nucleic Acids Research</i>, <i>47</i>(D1), D711‚ÄìD715. https://doi.org/10.1093/nar/gky964</span>


<a href="https://doi.org/10.1093/nar/gky964" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="nhmrc_management_2019">NHMRC, Council, A. R., &amp; Australia, U. (2019). <i>Management of Data and Information in Research: A guide supporting the Australian Code for the Responsible Conduct of Research</i> (Guide No.R41B; Number R41B). Commonwealth of Australia. https://www.nhmrc.gov.au/sites/default/files/documents/attachments/Management-of-Data-and-Information-in-Research.pdf</span>


<a href="https://www.nhmrc.gov.au/sites/default/files/documents/attachments/Management-of-Data-and-Information-in-Research.pdf" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cncb-ngdc_genome_2019">CNCB-NGDC. (2019). <i>Genome Sequence Archive for Human - Policies</i>. https://ngdc.cncb.ac.cn/gsa-human/policy/policy.jsp#responsibilitiesSubmitter</span>


<a href="https://ngdc.cncb.ac.cn/gsa-human/policy/policy.jsp#responsibilitiesSubmitter" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="band_me_2019">Band, G. (2019). <i>Me vs. the EGA part 2: uploading data</i>. https://gavinband.github.io/bioinformatics/data/2019/05/02/Me_versus_the_European_Genome_Phenome_Archive_part_two.html</span>


<a href="https://gavinband.github.io/bioinformatics/data/2019/05/02/Me_versus_the_European_Genome_Phenome_Archive_part_two.html" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="band_me_2019-1">Band, G. (2019). <i>Me vs. EGA</i>. https://gavinband.github.io/bioinformatics/data/2019/05/01/Me_versus_the_European_Genome_Phenome_Archive.html</span>


<a href="https://gavinband.github.io/bioinformatics/data/2019/05/01/Me_versus_the_European_Genome_Phenome_Archive.html" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="band_me_2019-2">Band, G. (2019). <i>Me vs. the EGA part 3: winning</i>. https://gavinband.github.io/bioinformatics/data/2019/05/12/Me_versus_the_European_Genome_Phenome_Archive_part_three.html</span>


<a href="https://gavinband.github.io/bioinformatics/data/2019/05/12/Me_versus_the_European_Genome_Phenome_Archive_part_three.html" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="band_me_2019-3">Band, G. (2019). <i>Me vs. the EGA part 4: losing again</i>. https://gavinband.github.io/bioinformatics/data/2019/05/15/Me_versus_the_European_Genome_Phenome_Archive_part_four.html</span>


<a href="https://gavinband.github.io/bioinformatics/data/2019/05/15/Me_versus_the_European_Genome_Phenome_Archive_part_four.html" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="plos_one_data_2019">ONE, P. L. O. S. (2019). Data Availability. In <i>Data Availability \textbar PLOS ONE</i>. https://journals.plos.org/plosone/s/data-availability</span>


<a href="https://journals.plos.org/plosone/s/data-availability" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="thorogood_canada_2018">Thorogood, A. (2018). Canada: will privacy rules continue to favour open science? <i>Human Genetics</i>, <i>137</i>(8), 595‚Äì602. https://doi.org/10.1007/s00439-018-1905-0</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1905-0" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="knoppers_introduction_2018">Knoppers, B. M., &amp; Joly, Y. (2018). Introduction: the why and whither of genomic data sharing. <i>Human Genetics</i>, <i>137</i>(8), 569‚Äì574. https://doi.org/10.1007/s00439-018-1923-y</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1923-y" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="phillips_international_2018">Phillips, M. (2018). International data-sharing norms: from the OECD to the General Data Protection Regulation (GDPR). <i>Human Genetics</i>, <i>137</i>(8), 575‚Äì582. https://doi.org/10.1007/s00439-018-1919-7</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1919-7" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="taylor_united_2018">Taylor, M. J., Wallace, S. E., &amp; Prictor, M. (2018). United Kingdom: transfers of genomic data to third countries. <i>Human Genetics</i>, <i>137</i>(8), 637‚Äì645. https://doi.org/10.1007/s00439-018-1921-0</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1921-0" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="eckstein_australia_2018">Eckstein, L., Chalmers, D., Critchley, C., Jeanneret, R., McWhirter, R., Nielsen, J., Otlowski, M., &amp; Nicol, D. (2018). Australia: regulating genomic data sharing to promote public trust. <i>Human Genetics</i>, <i>137</i>(8), 583‚Äì591. https://doi.org/10.1007/s00439-018-1914-z</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1914-z" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="molnar-gabor_germany_2018">Moln√°r-G√°bor, F. (2018). Germany: a fair balance between scientific freedom and data subjects‚Äô rights? <i>Human Genetics</i>, <i>137</i>(8), 619‚Äì626. https://doi.org/10.1007/s00439-018-1912-1</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1912-1" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="linden_common_2018">Linden, M., Prochazka, M., Lappalainen, I., Bucik, D., Vyskocil, P., Kuba, M., Sil√©n, S., Belmann, P., Sczyrba, A., Newhouse, S., Matyska, L., &amp; Nyr√∂nen, T. (2018). Common ELIXIR service for researcher authentication and authorisation. <i>F1000Research</i>, <i>7</i>. https://doi.org/10.12688/f1000research.15161.1</span>


<a href="http://dx.doi.org/10.12688/f1000research.15161.1" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="townend_conclusion_2018">Townend, D. (2018). Conclusion: harmonisation in genomic and health data sharing for research: an impossible dream? <i>Human Genetics</i>, <i>137</i>(8), 657‚Äì664. https://doi.org/10.1007/s00439-018-1924-x</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1924-x" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="majumder_united_2018">Majumder, M. A. (2018). United States: law and policy concerning transfer of genomic data to third countries. <i>Human Genetics</i>, <i>137</i>(8), 647‚Äì655. https://doi.org/10.1007/s00439-018-1917-9</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1917-9" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="kim_south_2018">Kim, H., Kim, S. Y., &amp; Joly, Y. (2018). South Korea: in the midst of a privacy reform centered on data sharing. <i>Human Genetics</i>, <i>137</i>(8), 627‚Äì635. https://doi.org/10.1007/s00439-018-1920-1</span>


<a href="http://dx.doi.org/10.1007/s00439-018-1920-1" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="woolley_responsible_2018">Woolley, J. P., Kirby, E., Leslie, J., Jeanson, F., Cabili, M. N., Rushton, G., Hazard, J. G., Ladas, V., Veal, C. D., Gibson, S. J., Tass√©, A.-M., Dyke, S. O. M., Gaff, C., Thorogood, A., Knoppers, B. M., Wilbanks, J., &amp; Brookes, A. J. (2018). Responsible sharing of biomedical data and biospecimens via the "Automatable Discovery and Access Matrix" (ADA-M). <i>NPJ Genomic Medicine</i>, <i>3</i>, 17. https://doi.org/10.1038/s41525-018-0057-4</span>


<a href="http://dx.doi.org/10.1038/s41525-018-0057-4" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="zappia_exploring_2018">Zappia, L., Phipson, B., &amp; Oshlack, A. (2018). Exploring the single-cell RNA-seq analysis landscape with the scRNA-tools database. <i>PLOS Computational Biology</i>, <i>14</i>(6), e1006245. https://doi.org/10.1371/journal.pcbi.1006245</span>


<a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006245" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="health_legal_genomic_2018">Health Legal. (2018). <i>Genomic Data &amp; Privacy Law: A summary of Health Legal‚Äôs report for Australian Genomics</i>. https://www.australiangenomics.org.au/wp-content/uploads/2021/09/Summary-Health-Legal-Report.pdf</span>


<a href="https://www.australiangenomics.org.au/wp-content/uploads/2021/09/Summary-Health-Legal-Report.pdf" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="sarkans_biostudies_2018">Sarkans, U., Gostev, M., Athar, A., Behrangi, E., Melnichuk, O., Ali, A., Minguet, J., Rada, J. C., Snow, C., Tikhonov, A., Brazma, A., &amp; McEntyre, J. (2018). The BioStudies database‚Äîone stop shop for all data supporting a life sciences study. <i>Nucleic Acids Research</i>, <i>46</i>(D1), D1266‚ÄìD1270. https://doi.org/10.1093/nar/gkx965</span>


<a href="https://doi.org/10.1093/nar/gkx965" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="nhmrc_australian_2018">NHMRC. (2018). <i>Australian Code for the Responsible Conduct of Research, 2018 \textbar NHMRC</i>. https://www.nhmrc.gov.au/about-us/publications/australian-code-responsible-conduct-research-2018</span>


<a href="https://www.nhmrc.gov.au/about-us/publications/australian-code-responsible-conduct-research-2018" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="nhmrc_national_2018">NHMRC. (2018). <i>National Statement on Ethical Conduct in Human Research (2007) - Updated 2018 \textbar NHMRC</i>. https://www.nhmrc.gov.au/about-us/publications/national-statement-ethical-conduct-human-research-2007-updated-2018</span>


<a href="https://www.nhmrc.gov.au/about-us/publications/national-statement-ethical-conduct-human-research-2007-updated-2018" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="perez-riverol_discovering_2017">Perez-Riverol, Y., Bai, M., da Veiga Leprevost, F., Squizzato, S., Park, Y. M., Haug, K., Carroll, A. J., Spalding, D., Paschall, J., Wang, M., del-Toro, N., Ternent, T., Zhang, P., Buso, N., Bandeira, N., Deutsch, E. W., Campbell, D. S., Beavis, R. C., Salek, R. M., ‚Ä¶ Hermjakob, H. (2017). Discovering and linking public omics data sets using the Omics Discovery Index. <i>Nature Biotechnology</i>, <i>35</i>(5), 406‚Äì409. https://doi.org/10.1038/nbt.3790</span>


<a href="https://www.nature.com/articles/nbt.3790" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="wang_gsa_2017">Wang, Y., Song, F., Zhu, J., Zhang, S., Yang, Y., Chen, T., Tang, B., Dong, L., Ding, N., Zhang, Q., Bai, Z., Dong, X., Chen, H., Sun, M., Zhai, S., Sun, Y., Yu, L., Lan, L., Xiao, J., ‚Ä¶ Zhao, W. (2017). GSA: Genome Sequence Archive*. <i>Genomics, Proteomics &amp; Bioinformatics</i>, <i>15</i>(1), 14‚Äì18. https://doi.org/10.1016/j.gpb.2017.01.001</span>


<a href="https://www.sciencedirect.com/science/article/pii/S1672022917300025" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="streeter_human-induced_2017">Streeter, I., Harrison, P. W., Faulconbridge, A., Flicek, P., Parkinson, H., &amp; Clarke, L. (2017). The human-induced pluripotent stem cell initiative‚Äîdata resources for cellular genetics. <i>Nucleic Acids Research</i>, <i>45</i>(Database issue), D691‚ÄìD697. https://doi.org/10.1093/nar/gkw928</span>


<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5210631/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="wellcome_trust_data_2017">Trust, W. (2017). Data, software and materials management and sharing policy - Grant Funding. In <i>Wellcome</i>. https://wellcome.org/grant-funding/guidance/data-software-materials-management-and-sharing-policy</span>


<a href="https://wellcome.org/grant-funding/guidance/data-software-materials-management-and-sharing-policy" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="grossman_case_2016">Grossman, R. L., Heath, A., Murphy, M., Patterson, M., &amp; Wells, W. (2016). A case for data commons: toward data science as a service. <i>Computing in Science &amp; Engineering</i>, <i>18</i>(5), 10‚Äì20. https://doi.org/10.1109/MCSE.2016.92</span>


<a href="http://ieeexplore.ieee.org/document/7548983/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="tucker_protecting_2016">Tucker, K., Branson, J., Dilleen, M., Hollis, S., Loughlin, P., Nixon, M. J., &amp; Williams, Z. (2016). Protecting patient privacy when sharing patient-level data from clinical trials. <i>BMC Medical Research Methodology</i>, <i>16 Suppl 1</i>, 77. https://doi.org/10.1186/s12874-016-0169-4</span>


<a href="http://dx.doi.org/10.1186/s12874-016-0169-4" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="the_global_alliance_for_genomics_and_health_federated_2016">THE GLOBAL ALLIANCE FOR GENOMICS AND HEALTH. (2016). A federated ecosystem for sharing genomic, clinical data. <i>Science</i>, <i>352</i>(6291), 1278‚Äì1280. https://doi.org/10.1126/science.aaf6162</span>


<a href="https://www.science.org/doi/10.1126/science.aaf6162" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="vayena_between_2016">Vayena, E., &amp; Gasser, U. (2016). Between openness and privacy in genomics. <i>PLoS Medicine</i>, <i>13</i>(1), e1001937. https://doi.org/10.1371/journal.pmed.1001937</span>


<a href="http://dx.doi.org/10.1371/journal.pmed.1001937" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="clough_gene_2016">Clough, E., &amp; Barrett, T. (2016). The Gene Expression Omnibus database. <i>Methods in Molecular Biology (Clifton, N.J.)</i>, <i>1418</i>, 93‚Äì110. https://doi.org/10.1007/978-1-4939-3578-9_5</span>


<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4944384/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="stephens_big_2015">Stephens, Z. D., Lee, S. Y., Faghri, F., Campbell, R. H., Zhai, C., Efron, M. J., Iyer, R., Schatz, M. C., Sinha, S., &amp; Robinson, G. E. (2015). Big data: astronomical or genomical? <i>PLoS Biology</i>, <i>13</i>(7), e1002195. https://doi.org/10.1371/journal.pbio.1002195</span>


<a href="http://dx.plos.org/10.1371/journal.pbio.1002195" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="knoppers_framework_2014">Knoppers, B. M. (2014). Framework for responsible sharing of genomic and health-related data. <i>The HUGO Journal</i>, <i>8</i>(1), 3. https://doi.org/10.1186/s11568-014-0003-1</span>


<a href="http://dx.doi.org/10.1186/s11568-014-0003-1" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="nih_nih_2014">NIH. (2014). <i>NIH Genomic Data Sharing Policy</i>. https://osp.od.nih.gov/wp-content/uploads/NIH_GDS_Policy.pdf</span>


<a href="https://osp.od.nih.gov/wp-content/uploads/NIH_GDS_Policy.pdf" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="cruk_data_2014">CRUK. (2014). Data sharing guidelines. In <i>Cancer Research UK</i>. https://www.cancerresearchuk.org/funding-for-researchers/applying-for-funding/policies-that-affect-your-grant/submission-of-a-data-sharing-and-preservation-strategy/data-sharing-guidelines</span>


<a href="https://www.cancerresearchuk.org/funding-for-researchers/applying-for-funding/policies-that-affect-your-grant/submission-of-a-data-sharing-and-preservation-strategy/data-sharing-guidelines" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="tryka_ncbis_2014">Tryka, K. A., Hao, L., Sturcke, A., Jin, Y., Wang, Z. Y., Ziyabari, L., Lee, M., Popova, N., Sharopova, N., Kimura, M., &amp; Feolo, M. (2014). NCBI‚Äôs Database of Genotypes and Phenotypes: dbGaP. <i>Nucleic Acids Research</i>, <i>42</i>(Database issue), D975‚Äì9. https://doi.org/10.1093/nar/gkt1211</span>


<a href="http://dx.doi.org/10.1093/nar/gkt1211" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="anderson_standardized_2013">Anderson, H. V., Weintraub, W. S., Radford, M. J., Kremers, M. S., Roe, M. T., Shaw, R. E., Pinchotti, D. M., &amp; Tcheng, J. E. (2013). Standardized cardiovascular data for clinical research, registries, and patient care: a report from the Data Standards Workgroup of the National Cardiovascular Research Infrastructure project. <i>Journal of the American College of Cardiology</i>, <i>61</i>(18), 1835‚Äì1846. https://doi.org/10.1016/j.jacc.2012.12.047</span>


<a href="http://dx.doi.org/10.1016/j.jacc.2012.12.047" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="linden_resource_2013">Linden, M., Nyr√∂nen, T., &amp; Lappalainen, I. (2013). RESOURCE ENTITLEMENT MANAGEMENT SYSTEM. <i>Trans-European Research and Education Networking Association</i>.</span>


</li>
<li><span id="rights_ocr_guidance_2012">Rights (OCR), O. for C. (2012). Guidance Regarding Methods for De-identification of Protected Health Information in Accordance with the Health Insurance Portability and Accountability Act (HIPAA) Privacy Rule. In <i>HHS.gov</i>. https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html</span>


<a href="https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="williams_health_2010">Williams, E. D., Tapp, R. J., Magliano, D. J., Shaw, J. E., Zimmet, P. Z., &amp; Oldenburg, B. F. (2010). Health behaviours, socioeconomic status and diabetes incidence: the Australian Diabetes Obesity and Lifestyle Study (AusDiab). <i>Diabetologia</i>, <i>53</i>(12), 2538‚Äì2545. https://doi.org/10.1007/s00125-010-1888-4</span>


<a href="http://dx.doi.org/10.1007/s00125-010-1888-4" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="kauffmann_importing_2009">Kauffmann, A., Rayner, T. F., Parkinson, H., Kapushesky, M., Lukk, M., Brazma, A., &amp; Huber, W. (2009). Importing ArrayExpress datasets into R/Bioconductor. <i>Bioinformatics</i>, <i>25</i>(16), 2092‚Äì2094. https://doi.org/10.1093/bioinformatics/btp354</span>


<a href="https://doi.org/10.1093/bioinformatics/btp354" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="kaye_data_2009">Kaye, J., Heeney, C., Hawkins, N., de Vries, J., &amp; Boddington, P. (2009). Data sharing in genomics‚Äìre-shaping scientific practice. <i>Nature Reviews. Genetics</i>, <i>10</i>(5), 331‚Äì335. https://doi.org/10.1038/nrg2573</span>


<a href="http://dx.doi.org/10.1038/nrg2573" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="rights_ocr_hipaa_2008">Rights (OCR), O. for C. (2008). The HIPAA Privacy Rule. In <i>HHS.gov</i>. https://www.hhs.gov/hipaa/for-professionals/privacy/index.html</span>


<a href="https://www.hhs.gov/hipaa/for-professionals/privacy/index.html" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="barr_risk_2007">Barr, E. L. M., Zimmet, P. Z., Welborn, T. A., Jolley, D., Magliano, D. J., Dunstan, D. W., Cameron, A. J., Dwyer, T., Taylor, H. R., Tonkin, A. M., Wong, T. Y., McNeil, J., &amp; Shaw, J. E. (2007). Risk of cardiovascular and all-cause mortality in individuals with diabetes mellitus, impaired fasting glucose, and impaired glucose tolerance: the Australian Diabetes, Obesity, and Lifestyle Study (AusDiab). <i>Circulation</i>, <i>116</i>(2), 151‚Äì157. https://doi.org/10.1161/CIRCULATIONAHA.106.685628</span>


<a href="http://dx.doi.org/10.1161/CIRCULATIONAHA.106.685628" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="davis_geoquery_2007">Davis, S., &amp; Meltzer, P. S. (2007). GEOquery: a bridge between the Gene Expression Omnibus (GEO) and BioConductor. <i>Bioinformatics</i>, <i>23</i>(14), 1846‚Äì1847. https://doi.org/10.1093/bioinformatics/btm254</span>


<a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btm254" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="parkinson_arrayexpresspublic_2007">Parkinson, H., Kapushesky, M., Shojatalab, M., Abeygunawardena, N., Coulson, R., Farne, A., Holloway, E., Kolesnykov, N., Lilja, P., Lukk, M., Mani, R., Rayner, T., Sharma, A., William, E., Sarkans, U., &amp; Brazma, A. (2007). ArrayExpress‚Äîa public database of microarray experiments and gene expression profiles. <i>Nucleic Acids Research</i>, <i>35</i>(suppl_1), D747‚ÄìD750. https://doi.org/10.1093/nar/gkl995</span>


<a href="https://doi.org/10.1093/nar/gkl995" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="rayner_simple_2006">Rayner, T. F., Rocca-Serra, P., Spellman, P. T., Causton, H. C., Farne, A., Holloway, E., Irizarry, R. A., Liu, J., Maier, D. S., Miller, M., Petersen, K., Quackenbush, J., Sherlock, G., Stoeckert, C. J., White, J., Whetzel, P. L., Wymore, F., Parkinson, H., Sarkans, U., ‚Ä¶ Brazma, A. (2006). A simple spreadsheet-based, MIAME-supportive format for microarray data: MAGE-TAB. <i>BMC Bioinformatics</i>, <i>7</i>(1), 489. https://doi.org/10.1186/1471-2105-7-489</span>


<a href="https://doi.org/10.1186/1471-2105-7-489" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="bard_ontologies_2004">Bard, J. B. L., &amp; Rhee, S. Y. (2004). Ontologies in biology: design, applications and future challenges. <i>Nature Reviews. Genetics</i>, <i>5</i>(3), 213‚Äì222. https://doi.org/10.1038/nrg1295</span>


<a href="http://dx.doi.org/10.1038/nrg1295" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="alrc_essentially_2003">ALRC. (2003). Essentially Yours: The Protection of Human Genetic Information in Australia (ALRC Report 96). In <i>ALRC</i>. https://www.alrc.gov.au/publication/essentially-yours-the-protection-of-human-genetic-information-in-australia-alrc-report-96/</span>


<a href="https://www.alrc.gov.au/publication/essentially-yours-the-protection-of-human-genetic-information-in-australia-alrc-report-96/" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li>
<li><span id="brunak_nucleotide_2002">Brunak, S., Danchin, A., Hattori, M., Nakamura, H., Shinozaki, K., Matise, T., &amp; Preuss, D. (2002). Nucleotide Sequence Database Policies. <i>Science</i>, <i>298</i>(5597), 1333‚Äì1333. https://doi.org/10.1126/science.298.5597.1333b</span>


<a href="https://www.science.org/doi/10.1126/science.298.5597.1333b" target="_blank"><i class="fa-solid fa-arrow-up-right-from-square"></i></a>

</li></ol>


    
    

    

    

</main>


            </div>
        </div>
    </div>
    <button id="back-to-top" class="btn btn-primary" type="button" aria-hidden="true" onclick="topFunction()">
        <i class="fa-solid fa-arrow-up"></i><span class="ms-2 d-none d-sm-inline">Back to top</span>
    </button>
    <footer id="footer">
    <div class="container py-1 py-4">
        <div class="row g-4 d-flex justify-content-between">
            <div class="col-lg-2 col-sm-6 d-flex justify-content-center">
                <img class="my-auto" width="88px" src="/human-omics-data-sharing-field-guide/assets/img/main_logo.png" alt="Australian BioCommons Logo" />
            </div>
            <div class="col-lg-3 col-sm-6">
                <ul class="mb-0">
                    <li><a href="/human-omics-data-sharing-field-guide/">Home</a></li>
                </ul>
            </div>
        </div>
        <div class="row">
            <div class="text-center pt-4 mb-0">
                <p>Icons used on this page from <a href="https://thenounproject.com/">The Noun Project</a> see <a href="/acknowledgements">Acknowledgements</a> for more info.</p>

            </div>
        </div>
    </div>
    <div class="copyright text-center p-4">
        <p>The content of this website is licensed under <a href="http://creativecommons.org/licenses/by/4.0/"><img src="https://i.creativecommons.org/l/by/4.0/88x31.png" alt="Creative Commons License" /></a>. The <a href="https://www.biocommons.org.au/">Australian BioCommons</a> is supported by <a href="https://www.bioplatforms.com/">Bioplatforms Australia</a>. Bioplatforms Australia is enabled by <a href="https://www.education.gov.au/national-collaborative-research-infrastructure-strategy-ncris">NCRIS</a>.  <a href="https://www.biocommons.org.au/privacy">Privacy</a>. <a href="https://www.biocommons.org.au/cookies">Cookies</a></p>

    </div>
</footer>

    

</body>
</html>
